File: Assignment_6/program2.java
import java.util.Scanner;


class Check
{

    public void CheckDigit(int iNo)
    {
        switch(iNo)

        {
            case 1:
            System.out.println("One");
            break;

            case 2:
            System.out.println("Two");
            break;

            case 3:
            System.out.println("Three");
            break;

            case 4:
            System.out.println("Four");
            break;

            case 5:
            System.out.println("Five");
            break;

            case 6:
            System.out.println("Six");
            break;

            case 7:
            System.out.println("Seven");
            break;

            case 8:
            System.out.println("Eight");
            break;

            case 9:
            System.out.println("Nine");
            break;

            case 0:
            System.out.println("Zero");
            break;

            default:
            System.out.println("Enter valid number.");

            

        }
    }
}
class program2 
{
    public static void main(String A[])
    {

        int iValue = 0;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the number :");
        iValue = sobj.nextInt();

        Check cobj = new Check();

        cobj.CheckDigit(iValue);




    }
    
}

File: Assignment_4/program2.c
#include<stdio.h>

void FactRev(int iNo)
{

    if(iNo < 0)
    {
        iNo = -iNo;
    }

    int iCnt = 0;
    

    for(iCnt = iNo; iCnt > 0; iCnt--)
    {
        if((iNo)%(iCnt)==0)
        {
            if((iNo)==(iCnt))
        {
            continue;

        }
            printf("%d\t",iCnt);
            
        }
    }

}

int main()
{

    int ivalue = 0;
   

    printf("Enter the Number : ");
    scanf("%d",&ivalue);

    FactRev(ivalue);

}